### PR TITLE
Move FFS example

### PR DESCRIPTION
**Pull request by InnocentBug**
I saw that the FFS example was misplaced.
Moved it to OpenMM and added it to automated testing.

### CODE REVIEW

**Comment by InnocentBug** [comment]
@pabloferz @gustavor101 the FFS fails with an actual problem.
Are masses not part of the snapshot right now?

**Comment by pabloferz** [comment]
This worked when initially implemented. I'll look into what could have caused the regression.

File: pysages/methods/ffs.py [hunk 1]
```diff
@@ -346,7 +347,7 @@ def running_window(grid, step, old_snapshots, run, sampler, helpers, cv):
                     has_conf_stored = True
 
     if success == 0:
-        raise Exception(f"Error in window {step}\n")
+        raise warn(f"Unable to estimate probability, exiting early at window {step}\n")
```
Comment (pabloferz) [inline pysages/methods/ffs.py:350]:
```suggestion
        warn(f"Unable to estimate probability, exiting early at window {step}\n")
```

File: pysages/methods/ffs.py [hunk 2]
```diff
@@ -346,7 +347,7 @@ def running_window(grid, step, old_snapshots, run, sampler, helpers, cv):
                     has_conf_stored = True
 
     if success == 0:
-        raise Exception(f"Error in window {step}\n")
+        warn(f"Unable to estimate probability, exiting early at window {step}\n")
```
Comment (pabloferz) [inline pysages/methods/ffs.py:350]:
```suggestion
        warn(f"Unable to estimate probability, exiting early at window {step}\n")
        return
```

File: examples/openmm/forward_flux_sampling.py [hunk 3]
```diff
@@ -58,14 +59,35 @@ def generate_simulation(pdb_filename=adp_pdb, T=T, dt=dt):
 
 # %%
 def main():
+    available_args = [
+        ("timesteps", "t", int, 1e9, "Max number of timesteps."),
+        ("cv-distance", "d", float, 50, "Distance from the intial (100) to the final dihedral."),
+        ("window-number", "Nw", int, 4, "Number of windows."),
+        ("sampling-steps", "S", inkkt, 20000, "Period for sampling configurations in the basin."),
```
Comment (pabloferz) [inline examples/openmm/forward_flux_sampling.py:66]:
```suggestion
        ("sampling-steps", "S", int, 20000, "Period for sampling configurations in the basin."),
```

File: .github/workflows/docker-ci.yml [hunk 4]
```diff
@@ -109,3 +109,22 @@ jobs:
...
+        name: Download artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: pysages
+          path: /tmp
+      -
+        name: Load and run test
+        run: |
+          docker load --input /tmp/pysages.tar
+          docker run -v /tmp:/tmp -t pysages bash -c "cd PySAGES/examples/openmm/ && python3 forward_flux_sampling.py --cv-distance=1 --sampling-steps=200"
```
Comment (pabloferz) [inline .github/workflows/docker-ci.yml:130]:
```suggestion
          docker run -v /tmp:/tmp -t pysages bash -c "cd PySAGES/examples/openmm/ && python3 forward_flux_sampling.py --cv-distance=1 --sampling-steps=2 --replicas=2"
```

**Review by pabloferz — APPROVED** [review]
@gustavor101 suggested changing the example to ADP without solvent, the starting basing, and the parameters. It is now running in around 15 minutes which I deemed acceptable.

**Comment by InnocentBug** [comment]
Sounds good.
Let's merge

**Pull request merged**